caramel/config.py

```diff
@@ -23,6 +23,20 @@ def add_inifile_argument(parser, env=None):
     )
 
 
+def add_ca_arguments(parser):
+    """Adds a ca-cert and ca-key argument to a given parser"""
+    parser.add_argument(
+        "--ca-cert",
```

Comment:
Here it's ok to use "ca-cert" as it is what the user expects.  It reads right on the command line.